tachiyomiorg/tachiyomi-extensions
# NewToki/ManaToki: update to v1.3.24

- Refactor + cleanups
- Update domain number automatically using redirections and Telegram channel
- Updated fallback domain number pattern
- Better preferences and filters
- Latest updates page + manga title check
- Manga URL check
- Removed `getUrlWithoutDomainWithFallback()` as the site no longer include brackets in URLs
  - See #5935
- Strings localized by [**@moonji**](https://hosted.weblate.org/user/moonji/) and [**@manatails**](https://hosted.weblate.org/user/manatails/)
- Marked as NSFW
- Remove manga title from chapter name

This should:
- Fix #11558
- Fix #12811

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
